github/azure-self-hosted-runners/README.md

```diff
@@ -0,0 +1,64 @@
+# Self-hosted Github Runners with Garm
+
+Those instructions deploy Garm for self-hosted github runners on azure (c)VMs. Garm listens to webhooks from github and spawns VMs on demand.
+
+## Build
+
+```bash
+docker build -t garm-azure .
+```
+
+## Service
+
+The application is deployed as an [ACI](https://azure.microsoft.com/en-us/products/container-instances) Container Group. It is bootstrapped with an init container which creates an admin user, an initial repository registration, and a pool for the repository. It keeps database state on a Storage Account-backed volume. The app will expose an HTTP/S ports for github webhooks.
```

Comment:
admin user for garm?

Reply:
The original idea was to not have a user that interacts with the db, but configure the whole thing declarative in the init script . Every new revision would then tear down everything and recreate everything.

I'm don't think this is a good idea anymore: there might be dangling resources (e.g. a runner is currently running a workflow and its resource group doesn't get garbage collected after a redeployment)

So what we could do instead:

remove the repo initialisation from the init container and give instructions to create repos + pools interactively. The state is kept in the storage account and survives redeployments. wdyt?